src/app/pages/instances.tsx

```diff
@@ -56,6 +59,30 @@ export const InstancesPage: FunctionComponent<InstancesPageProps> = ({}: Instanc
         });
     };
 
+    const doDeleteInstance = (): void => {
+        if (!instanceToDelete) return;
+        setLoading(true);
```

Comment:
No need to set the loading state variable here.  That's intended for when the registries are initially loaded on first page access.